shared/src/Datadog.Trace.ClrProfiler.Native/runtimeid_store.cpp

```diff
@@ -13,7 +13,22 @@ RuntimeIdStore::RuntimeIdStore() : RuntimeIdStore(IsRunningOnIIS())
 RuntimeIdStore::RuntimeIdStore(bool isIis) :
     m_isIis{isIis}
 {
-    m_process_runtime_id = ::shared::GenerateRuntimeId();
+    if (isIis)
+    {
+        m_process_runtime_id = ::shared::GenerateRuntimeId();
+    }
+    else
+    {
+        const auto internalRuntimeId = ::shared::GetEnvironmentValue(EnvironmentVariables::InternalRuntimeId);
```

Comment:
In order to make CodeHotspot work, Profiles and Traces must have the same RuntimeId. Because benchmarks runs out of process we need to force the same RuntimeId as the parent process ( the one sending the spans).